### PR TITLE
[DO-1906] Add_ukrainian_language_search_country_of_registration

### PR DESCRIPTION
**Pull request by Iva-khar**
Added `country_of_registration_uk` filter

### CODE REVIEW

**Review by EmmaSamoilova — APPROVED** [review]

File: business_register/filters.py [hunk 1]
```diff
@@ -353,7 +353,13 @@ class CompanySanctionFilterSet(BaseSanctionFilter):
...
         distinct=True,
-        help_text='Filter by country_of_registration of company. Type: case insensitive string contains',
+        help_text='Filter by country_of_registration of company in English. Type: case insensitive string contains',
+    )
+    country_of_registration_uk = filters.CharFilter(
+        field_name='country_of_registration__name_uk',
+        lookup_expr='icontains',
+        distinct=True,
+        help_text='Filter by country_of_registration of company in Ukrainian. Type: case insensitive string contains',
     )
```
Comment (roman-tiukh) [inline business_register/filters.py:363]: We talked about the fact that it should be one filter for both fields at once

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/filters.py (hunk 1 above)
Comment (Iva-khar) [reply business_register/filters.py]: done

File: business_register/views/sanction_views.py [hunk 2]
```diff
@@ -67,5 +67,6 @@ class CompanySanctionViewSet(RegisterViewMixin,
         'taxpayer_number',
         'address',
         'types_of_sanctions__name',
-        'country_of_registration__name'
+        'country_of_registration__name',
+        'country_of_registration__name_uk',
```
Comment (roman-tiukh) [inline business_register/views/sanction_views.py:71]: here

File: business_register/filters.py [hunk 3]
```diff
@@ -350,16 +350,21 @@ class CompanySanctionFilterSet(BaseSanctionFilter):
...
 
     class Meta:
         model = CompanySanction
         fields = {}
 
+    def filter_en_and_uk_fields(self, queryset, name, value):
+        return queryset.filter(
+            Q(country_of_registration__name__icontains=value) | Q(country_of_registration__name_uk__icontains=value)
+        )
+
```
Comment (roman-tiukh) [inline business_register/filters.py:367]: Meta should located at bottom of FilterSet

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/filters.py [hunk 4]
```diff
@@ -350,16 +350,21 @@ class CompanySanctionFilterSet(BaseSanctionFilter):
         help_text='Filter by additional info of company. Type: case insensitive string contains',
     )
     country_of_registration = filters.CharFilter(
-        field_name='country_of_registration__name',
-        lookup_expr='icontains',
+        method='filter_en_and_uk_fields',
```
Comment (roman-tiukh) [inline business_register/filters.py:353]: method needs to be named clearer

**Review by roman-tiukh — CHANGES_REQUESTED** [review]

File: business_register/views/sanction_views.py (hunk 2 above)
Comment (Iva-khar) [reply business_register/views/sanction_views.py:71]: ?
delete it?

File: business_register/filters.py (hunk 3 above)
Comment (Iva-khar) [reply business_register/filters.py:367]: done

File: business_register/filters.py (hunk 4 above)
Comment (Iva-khar) [reply business_register/filters.py]: done

**Review by roman-tiukh — APPROVED** [review]

**Pull request merged**